iothub/service/src/Configurations/Configuration.cs

```diff
@@ -41,19 +41,19 @@ internal Configuration()
         }
 
         /// <summary>
-        /// Gets the identifier for the configuration.
+        /// The unique identifier of the configuration.
```

Comment:
For all of the property changes I recall a while back that we had an analysis rule that flags properties as needing "Gets ..." "Sets ...", or "Gets or sets ..." depending on the property access modifiers. Did we remove that?

Maybe we need to utilize the value feild?

```csharp
/// <summary>
/// Gets the identifier for the configuration.
/// </summary>
/// <value>
/// The full identifier of the configuration as a string.
/// </value>
```

Reply:
I changed these to match the (what I thought were better) swagger comments.

I don't recall ever seeing a C# standard that properties need to say "Gets or sets...". Maybe that is a Java thing?